### DISCUSSION TITLE
File operations buttons row

### DISCUSSION THREAD

**Discussion by mikekazakov** [mikekazakov/nimble-commander#241] [Ideas]
Add a row of buttons in the bottom of a window, like in most of dual-pane file managers.
Problems:
- what to do with hotkeys redefinition?
- what to do with modifiers keys?
- what to do with overlapped terminal?

**Comment by iflowfor8hours** [comment]
Perhaps a first try at this would be a kind of learning mode that has the row of buttons either detached (in another floating window) to avoid having to make a decision about the overlapped terminal and make major changes to how that code works.
I'm sure you've thought about the key redefinition problem more than I have, but I would propose just ignoring changes to the major modifiers, and making this a UI-only change initially. Like FAR manager does.

nimble is the best OSX dual-pane file manager I've found, great work.

**Comment by VsevolodGolovanov** [comment]
Also want this one - I need to see these hints when holding Shift and other modifier keys. Especially since I'm new to MacOS and mucking with Karabiner.
